sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -74,32 +65,51 @@ export class InstitutionDto extends PartialType(CreateInstitutionDto) {
   @IsOptional()
   institutionTypeName: string;
 }
+export class InstitutionAddress {
+  addressLine1: string;
+  addressLine2: string;
+  city: string;
+  provinceState: string;
+  country: string;
+  postalCode: string;
+}
 
-export class AESTInstitutionDetailDto {
-  legalOperatingName: string;
+export class InstitutionContactDto {
```

Comment:
After the introduction of the swagger, I noticed many DTO as Dto. Since it is an acronym we should keep then always as upper case, right?
I know that maybe some DTOs here are from existing code, but if possible we can take some opportunities to change them 😉